src/backend/marsha/core/serializers/video.py

```diff
@@ -288,6 +295,22 @@ class InitLiveStateSerializer(serializers.Serializer):
     type = serializers.ChoiceField(LIVE_TYPE_CHOICES)
 
 
+class LivePairingSerializer(serializers.ModelSerializer):
+    """Serializer for LivePairing model."""
+
+    class Meta:  # noqa
+        model = LivePairing
+        fields = ("secret",)
+        read_only_fields = ("secret",)
```

Comment:
You should add a test trying to patch or put the object modifyng the secret field